### DISCUSSION TITLE
[Shrink] Quick resize method using mouse only

### DISCUSSION THREAD

**Discussion by SanshooSenshi** [WindowTop/WindowTop-App#170] [Ideas]
It could be awesome to change size of a shrinked window on holding right click and slide left ( bigger ) slide right ( smaller ).  ( And if that move just happened, highlight the interact icon for few seconds to make it quickly set up. )

**Comment by gileli121** [comment]
@SanshooSenshi 
Thanks for the suggestion.
By saying "slide right" and "slide left" you mean to use the mouse wheel?

**Reply by SanshooSenshi** [reply]
oh that's a possibility indeed, I was thinking about moving the mouse right or left ...

**Reply by gileli121** [reply]
I suggest that will be a key that while you hold it and scroll up, the shrink will become bigger as you scroll up.
And when you scroll down, the shrink will become smaller.

The key that you need to hold will be on the keyboard

**Reply by SanshooSenshi** [reply]
I think I won't use it as much as if I can do the all thing on mouse only !

**Reply by gileli121** [reply]
@SanshooSenshi 
I understand what you mean.
You mean to hold the right mouse key and while holding and moving right, it will resize it to bigger, and while moving left it will resize smaller.
It is a very smart idea since you don't even need the keyboard.

I don't understand why I did not get you before.
Maybe I will develop it one day but it should also come with some "Tips" window that shows it.. otherwise, no one will know about it.
